core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -237,6 +241,18 @@ private[data] trait KleisliCommutativeArrow[F[_]] extends CommutativeArrow[Kleis
   implicit def F: CommutativeMonad[F]
 }
 
+private[data] trait KleisliArrowChoice[F[_]] extends ArrowChoice[Kleisli[F, ?, ?]] with KleisliArrow[F] {
```

Comment:
We can fold `KleisliArrow` into `KleisliArrowChoice` right? `KleisliCommutativeArrow` can extend `KleisliArrowChoice` .